src/Microsoft.AspNet.Diagnostics.Elm/project.json

```diff
@@ -17,5 +16,6 @@
                 "System.Threading": "4.0.11-beta-*"
             }
         }
-    }
+    },
+    "resource": ["Views/*.cshtml"]
```

Comment:
Ah weird. Well, `compileExclude` is I think the right thing to do here. I wouldn't rename the file extension because they *are* in fact real Razor files (they use Razor syntax, after all). The fact that they're used for non-MVC scenarios is separate.